conf/app.routes

```diff
@@ -41,10 +41,10 @@ POST        /:journeyId/check-your-answers-vat        uk.gov.hmrc.claimvatenrolm
 GET         /can-not-confirm-business                 uk.gov.hmrc.claimvatenrolmentfrontend.controllers.errorPages.KnownFactsMismatchController.show()
 
 # Invalid Account Type Error
-GET         /can-not-use-this-service                 uk.gov.hmrc.claimvatenrolmentfrontend.controllers.errorPages.InvalidAccountTypeController.show()
+GET         /error/not-authorised                     uk.gov.hmrc.claimvatenrolmentfrontend.controllers.errorPages.InvalidAccountTypeController.show()
 
-# Unauthorised User Error
-GET         /can-not-use-this-service-2               uk.gov.hmrc.claimvatenrolmentfrontend.controllers.errorPages.UnmatchedUserErrorController.show()
+# Unmatched User Error
+GET         /error/not-enrolled                       uk.gov.hmrc.claimvatenrolmentfrontend.controllers.errorPages.UnmatchedUserErrorController.show()
 
 # Enrolment Already Allocated Error
- GET         /can-not-use-this-service-3              uk.gov.hmrc.claimvatenrolmentfrontend.controllers.errorPages.EnrolmentAlreadyAllocatedController.show()
+ GET        /error/already-enrolled                   uk.gov.hmrc.claimvatenrolmentfrontend.controllers.errorPages.EnrolmentAlreadyAllocatedController.show()
```

Comment:
Might be worth creating another routes file to separate out the error pages if we'll be adding a lot more pages in the future